src/field/pokemon-sprite-sparkle-handler.ts

```diff
@@ -26,10 +26,10 @@ export default class PokemonSpriteSparkleHandler {
       if (!s.pipelineData["teraColor"] || !(s.pipelineData["teraColor"] as number[]).find((c) => c)) {
         continue;
       }
-      if (!s.visible || (s.parentContainer instanceof Pokemon && !s.parentContainer.parentContainer)) {
+      if (!s.visible || (s.parentContainer.type === "pokemon" && !s.parentContainer.parentContainer)) {
```

Comment:
```suggestion
      if (!s.visible || (s.parentContainer.type === "Pokemon" && !s.parentContainer.parentContainer)) {
```
Is this case sensitive?

Reply:
Yes indeed